docs/troubleshooting.md

```diff
@@ -0,0 +1,199 @@
+
+$ oc describe servicebroker
+Name:           ansible-service-broker
+Namespace:
+Labels:         <none>
+Annotations:    openshift.io/generated-by=OpenShiftNewApp
+API Version:    servicecatalog.k8s.io/v1alpha1
+Kind:           ServiceBroker
+Metadata:
+  Creation Timestamp:   2017-10-05T17:21:01Z
+  Finalizers:
+    kubernetes-incubator/service-catalog
+  Generation:           1
+  Resource Version:     21
+  Self Link:            /apis/servicecatalog.k8s.io/v1alpha1/servicebrokers/ansible-service-broker
+  UID:                  8f3782b5-a9f1-11e7-ab6a-0242ac110006
+Spec:
+  Auth Info:
+    Bearer:
+      Secret Ref:
+        Kind:           Secret
+        Name:           ansibleservicebroker-client
+        Namespace:      ansible-service-broker
+  Relist Behavior:      Duration
+  Relist Duration:      15m0s
+  Relist Requests:      0
+  URL:                  https://asb.ansible-service-broker.svc:1338/ansible-service-broker/
+Status:
+  Conditions:
+    Last Transition Time:       2017-10-05T17:22:01Z
+    Message:                    Error fetching catalog. Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: x509: certificate signed by unknown authority
+    Reason:                     ErrorFetchingCatalog
+    Status:                     False
+    Type:                       Ready
+  Operation Start Time:         2017-10-05T17:22:02Z
+  Reconciled Generation:        0
+Events:
+  FirstSeen     LastSeen        Count   From                                    SubObjectPath   Type            Reason                  Message
+  ---------     --------        -----   ----                                    -------------   --------        ------                  -------
+  6s            6s              1       service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+  6s            2s              9       service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: x509: certificate signed by unknown authority
+```
+
+#### Resolution: Provide caBundle to service-catalog
+
+We can get the caBundle with the following command:
+
+```
+$ oc get secret -n kube-service-catalog -o go-template='{{ range .items }}{{ if eq .type "kubernetes.io/service-account-token" }}{{ index .data "service-ca.crt" }}{{end}}{{"\n"}}{{end}}' | tail -n 1
+LS0t...really long string...
+```
+
+And update the servicebroker object, adding `caBundle` to the ansible-service-broker's `Spec`:
```

Comment:
Once we have the `caBundle` we can update the servicebroker object, adding `caBundle` to the ansible-service-broker's `Spec`: